### PR TITLE
1.0/293 add app shell caching

### PR DESCRIPTION
**Pull request by charlesstrange2**
Magic plugin, "offline-plugin" - builds app-caches for webpack output (for old browsers) and instantiates a service worker cacheloader for new browsers. It's a bit magic but appears to be doing it's job in the console...

a second set of :eyes: would be useful.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -114,8 +114,8 @@ if (process.env.NODE_ENV === 'production') {
         new webpack.LoaderOptionsPlugin({
             minimize: true
         }),
-	new CopyWebpackPlugin([{
+	    new CopyWebpackPlugin([{
```
Comment (ob6160) [inline webpack.config.js]: Indentation ✂️

**Review by katjam — APPROVED** [review]
Apart from that one indent. :thermometer: 

What's the diff in adding ; to end of lines and not?

**Comment by charlesstrange2** [comment]
I refer ";" at the end of my javascipt statements. JS thinks it's optional (https://www.codecademy.com/blog/78) but I feel it adds clarity and better matches other languages I'm used to.

**Comment by katjam** [comment]
Good explain!! I thought it might be a performance thing... but I agree
with you 100%

**Pull request merged**